Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM fedora:32
+RUN dnf -y update
+RUN dnf -y install git
+RUN dnf -y groupinstall "Development Tools" "Development Libraries"
```

Comment:
This could be simplified to: `dnf --assumeyes --refresh install @development-tools @development-libs`

Reply:
LGTM